packages/compiler/core/checker.ts

```diff
@@ -972,13 +972,12 @@ export function createChecker(program: Program): Checker {
       return undefined;
     }
 
-    // we leave namespaces for interface members as undefined
     if (
       node.kind === SyntaxKind.OperationStatement &&
       node.parent &&
       node.parent.kind === SyntaxKind.InterfaceStatement
     ) {
-      return undefined;
+      return getParentNamespaceType(node.parent);
```

Comment:
We need to do that so that the model expressions can retrieve the namespace they belong to. When we migrate to versioned enum we could remove the entire namespace assignment for model expresions